### PR TITLE
ss/DCOS-55748 Adding new UI screenshots to Networking page.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-55748
## Description of changes being made

Adding new screenshots for Networking UI.

Still needed:
Screenshot for Networks> details pages, especially the tasks pages. These need to have actual data in them, rather than empty screens.

### CODE REVIEW

**Comment by SarahStegall** [comment]
I need values for the tables in /1.14/gui/networking/.

**Comment by SarahStegall** [comment]
Local build was successful and correct.

**Comment by SarahStegall** [comment]
I still need information for the Tasks and Service Account tables.

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 1]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+The Service Addresses tab displays a summary of the statistics for each network service address:
+
+![Service Addresses tab](/1.14/img/GUI-Networking-Service-Addresses-Main.png)
+
+Figure 3 - Service Addresses tab
+
+| Name | Description |
+|---------|--------------|
+| Name |   |
+| Successes |        |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: amount of successful requests

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 2]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+| Failures | |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: amount of failed requests

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 3]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+| Failure % | |
+| P99 Latency |   |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: P99 latency means that 99% of the requests should be faster than given latency. In other words only 1% of the requests are allowed to be slower.

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 4]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+| Container IP |        |
+| Port Mappings |        |
+
+## Details
+
+The **Details** tab lists the following information:
+
+| Name | Description |
+|---------|--------------|
+| Name |   |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: name of the virtual network

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 5]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+| IP Subnet |        |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: IP Subnet of the network
Note: it's like a mask in a well known format that should be familiar to operators https://en.m.wikipedia.org/wiki/Subnetwork

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 6]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+
+The Service Addresses tab displays a summary of the statistics for each network service address:
+
+![Service Addresses tab](/1.14/img/GUI-Networking-Service-Addresses-Main.png)
+
+Figure 3 - Service Addresses tab
+
+| Name | Description |
+|---------|--------------|
+| Name |   |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: I believe it's Mesos DNS Name

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 7]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+
+Figure 2 - Networks detail
+
+## Tasks
+
+The **Tasks** tab lists the following information:
+
+| Name | Description |
+|---------|--------------|
+| Task |   |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: task name

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 8]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+| Container IP |        |
+| Port Mappings |        |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: Host port to container port mapping if exists

File: pages/mesosphere/dcos/1.14/gui/networking/index.md [hunk 9]
```diff
@@ -8,8 +8,80 @@ render: mustache
...
+Figure 2 - Networks detail
+
+## Tasks
+
+The **Tasks** tab lists the following information:
+
+| Name | Description |
+|---------|--------------|
+| Task |   |
+| Container IP |        |
```
Comment (nLight) [inline pages/mesosphere/dcos/1.14/gui/networking/index.md]: IP address of the container

**Comment by SarahStegall** [comment]
All changes accounted for. Local build was successful and correct.

**Pull request merged**